client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -35,8 +35,8 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = ( {
 								NO__( "Let's set up your website – it takes only a moment." ) }
 						</h2>
 						<StepperWizard>
-							<VerticalSelect />
-							{ ( siteVertical || siteTitle ) && <SiteTitle /> }
+							{ React.createElement( VerticalSelect ) }
+							{ ( siteVertical || siteTitle ) && React.createElement( SiteTitle ) }
```

Comment:
Sneaky! :joy: 

Maybe add a comment as to why we're doing this? :sweat_smile:

Reply:
I'm not in love with effectively side-stepping the type system 😬 

I'd still prefer a different solution, but this worked 🤷‍♂ 

We can add a comment.

Reply:
I took a different approach which seems much better. Rather than creating, cloning and recreating the components with additional props, we pass an array of the desired components and let the `StepperWizard` render them with the desired props.

Feels better and seems to be well typed.

Reply:
Okay, looks good! Let's go with that :+1: